### PR TITLE
Password protect partnered events

### PR DESCRIPTION
**Pull request by ash6851**
backend code for https://github.com/CoderDojoChi/coderdojochi/issues/402.  front end isn't pretty :)

### CODE REVIEW

File: coderdojochi/settings.py [hunk 1]
```diff
@@ -25,7 +26,7 @@
 SECRET_KEY = 'e^u3u$pukt$s=6#&9oi9&jj5ow6563fuka%y9t7i*2laalk^l$'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = str_to_bool(os.environ.get('DEBUG')) or False
+DEBUG = str_to_bool(os.environ.get('DEBUG')) and 'test' not in sys.argv or False
```
Comment (karbassi) [inline coderdojochi/settings.py]: @ash6851 Can you explain what this addition does?

Comment (ash6851) [inline coderdojochi/settings.py]: @karbassi when I was writing tests I was trying to check response url, but the redirect was being intercepted by the redirect debug page and my response url was wrong.  Let me know if you need more explanation

Comment (karbassi) [inline coderdojochi/settings.py]: @ash6851 ah, yes. This needs to be set to default to false. I'll commit a fix ASAP.

Comment (karbassi) [inline coderdojochi/settings.py]: @ash6851 Merge in #409

**Comment by karbassi** [comment]
This was merged in #424 since I couldn't commit my changes to your branch.

Github added a feature that solves this issue. On your next PR, make sure the **Allow edits from maintainers** is checked. 

screen shot 2016-10-06 at 7 57 36 pm

**Pull request closed without merging**